### PR TITLE
Flame: extracting segments with trans-coding 

### PR DESCRIPTION
**Pull request by jakubjezek001**
## Brief description
Publishing with on flight trans-coding 

## Description
Supporting any flame xml presets, either shared or build into installation or even at some custom directory. All is configurable per preset in OpenPype settings.

## Additional info
- also added one more attribute in Creator plugin for respecting segment's order index.

## Testing notes:
1. publish from your test publishing sequence

### CODE REVIEW

**Comment by mkolar** [comment]
Task linked: [OP-1537 Flame - Submitting jobs to Burner farm](https://app.clickup.com/t/6658547/OP-1537)

File: openpype/hosts/flame/api/render_utils.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+    if shared_preset_path:
+        return os.path.dirname(shared_preset_path)
+
+    # then try installed results
+    installed_preset_path = _validate_results(installed_results)
+
+    if installed_preset_path:
+        return os.path.dirname(installed_preset_path)
+
+    # if nothing found then return None
```
Comment (iLLiCiTiT) [inline openpype/hosts/flame/api/render_utils.py:124]: GBut you're returning `False` :)

**Comment by iLLiCiTiT** [comment]
Please double check the changes of plugin orders with hiero plugins.

**Review by iLLiCiTiT — APPROVED** [review]
If the plugins order were validated and hiero publishing works then I Approve :)

**Pull request merged**